lib/cocoapods-core/specification/linter.rb

```diff
@@ -337,6 +337,19 @@ def _validate_libraries(libs)
         end
       end
 
+      # Performs validations related to the `vendored_libraries` attribute.
+      #
+      # @param [Array<String>] vendored_libraries the values specified in the `vendored_libraries` attribute
+      #
+      def _validate_vendored_libraries(vendored_libraries)
+        vendored_libraries.each do |lib|
+          lib_name = lib.downcase
+          unless lib_name.end_with?('.a') && lib_name.start_with?('lib')
```

Comment:
@amorde it seems like you are checking full file name (with path) instead of just basename like in the line below.